measure_extinction/extdata.py

```diff
@@ -1091,19 +1098,16 @@ def plot(
             fontsize for plot
         """
         if alax:
-            # compute A(V) if it is not available
-            if "AV" not in self.columns.keys():
-                self.trans_elv_alav()
-            av = _get_column_val(self.columns["AV"])
-            if self.type_rel_band != "V":  # not sure if this works (where is RV given?)
-                # use F04 model to convert AV to AX
-                rv = _get_column_val(self.columns["RV"])
-                emod = F04(rv)
-                (indx,) = np.where(self.type_rel_band == self.names["BAND"])
-                axav = emod(self.waves["BAND"][indx[0]])
-            else:
-                axav = 1.0
-            ax = axav * av
+            # transform the extinctions from E(lambda-V) to A(lambda)/A(V)
+            self.trans_elv_alav()
+
+            # the next part is not working. TODO: fix this
+            # if self.type_rel_band != "V":  # not sure if this works (where is RV given?)
+            #     # use F04 model to convert AV to AX
+            #     rv = _get_column_val(self.columns["RV"])
+            #     emod = F04(rv)
+            #     (indx,) = np.where(self.type_rel_band == self.names["BAND"])
+            #     axav = emod(self.waves["BAND"][indx[0]])
```

Comment:
Should be removed if not used.

Reply:
I am not using this, but I was unsure whether you are using it. I removed it for now, but if needed it can be added again later.